features/configuration_files/masking_smells.feature

```diff
@@ -53,9 +63,28 @@ Feature: Masking smells using config files
     """
 
   Scenario: Disable UtilityFunction for non-public methods
-    Given the smelly file 'smelly_with_modifiers.rb'
-    And a configuration file 'non_public_modifiers_mask.reek'
-    When I run reek -c non_public_modifiers_mask.reek smelly_with_modifiers.rb
+    Given a file named "smelly_with_modifiers.rb" with:
+      """
+      # Smelly class for testing purposes
+      #
+      # Not necessary for the feature per se but for
+      # removing distracting output. :reek:UnusedPrivateMethod
```

Comment:
The UnusedPrivateMethod detector is disabled by default, so this comment can be removed.

Reply:
Good catch! Removed.